.gitignore

```diff
@@ -103,3 +104,6 @@ Doc/*/*/hevea.sty
 
 #Ignore unittest cache dirctory
 .cache/
+
+#Ignore vscode directory
+.vscode
```

Comment:
Can you include a final line break here please?

Reply:
sure